dodiscover/ci/_script.py

```diff
@@ -0,0 +1,63 @@
+# type: ignore
+
+from math import frexp
+
+import numpy as np
+
+from dodiscover.ci import g_square_binary, g_square_discrete
+from dodiscover.ci.tests import testdata
+
+if __name__ == "__main__":
+
+    dm = np.array([testdata.bin_data]).reshape((5000, 5))
+    x = 0
+    y = 1
+
+    sets = [[], [2], [2, 3], [3, 4], [2, 3, 4]]
```

Comment:
Should this test script be put under the `tests/` folder? Is this something we expect users to run? It seems more likely that they'd use the conditional-independence classes.

Reply:
And if this is a test script, should it not use unittest or some alternative and assert some outcome is correct?